webextensions/api/declarativeNetRequest.json

```diff
@@ -0,0 +1,1656 @@
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/RuleCondition",
+            "support": {
+              "chrome": {
+                "version_added": "84"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": "15"
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "domains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84",
+                  "notes": "Deprecated in Chrome 101, use <code>initiatorDomains</code> instead."
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "84",
+                  "notes": "Deprecated in Chrome 101, use <code>excludedInitiatorDomains</code> instead."
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedInitiatorDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
+                "safari_ios": "mirror"
+              }
+            }
+          },
+          "excludedRequestDomains": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "101"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15"
```

Comment:
`excludedRequestDomains ` is not supported by shipping Safari. It is in Safari Technology Preview.

Reply:
```suggestion
                  "version_added": false
```